packages/stan/src/atom.js

```diff
@@ -0,0 +1,43 @@
+/** @typedef {import('./types').WPCommonAtomConfig} WPCommonAtomConfig */
+/**
+ * @template T
+ * @typedef {import("./types").WPAtom<T>} WPAtom
+ */
+
+/**
+ * Creates a basic atom.
+ *
+ * @template T
+ * @param {T}                   initialValue Initial value in the atom. *
+ * @param {WPCommonAtomConfig=} config       Common Atom config.
```

Comment:
```suggestion
 * @param {WPCommonAtomConfig=} config       Common atom's config.
```

Reply:
I think I'm going to skip this one as I believe the existing name is also good enough.